test/Unosquare.Swan.Test/JsonTest.cs

```diff
@@ -315,23 +307,20 @@ public void ObjectWithArrayWithData_ReturnsObjectWithArrayWithDataDeserialized()
         {
             var data = Json.Deserialize<ArrayJsonWithInitialData>("{\"Id\": 2,\"Properties\": [\"THREE\"]}");
 
-            Assert.IsNotNull(data);
             Assert.AreEqual(2, data.Id);
             Assert.AreEqual(1, data.Properties.Length);
         }
 
         [Test]
         public void WithJsonProperty_ReturnsPropertiesDeserialized()
         {
-            var obj = new JsonPropertySample() {Data = "OK", IgnoredData = "OK"};
+            var obj = new JsonPropertySample {Data = "OK", IgnoredData = "OK"};
             var data = Json.Serialize(obj);
```

Comment:
Are you testing Deserialize or Serialize method here?

Reply:
I will rewrite some tests on another branch.